### PR TITLE
[364] Added ability to set image tag for loaders

### PR DESCRIPTION
**Pull request by jaronoff97**
This PR changes the loader TF module to allow us to set the version for the image. By doing this, we will be able to make the image repositories immutable to prevent any image tags from getting overwritten. That work will be covered in a follow up PR so as to isolate the changes being made.

### CODE REVIEW

**Comment by jaronoff97** [comment]
The TF diff has to change the task definitions for all of the loaders, so we see thirteen changes in all.

**Comment by Mr0grog** [comment]
Rebased so this merges cleanly.

File: terraform/modules/loader/main.tf [hunk 1]
```diff
@@ -2,7 +2,7 @@ module "loader_task" {
   source = "../task"
 
   name    = var.name
-  image   = "681497372638.dkr.ecr.us-west-2.amazonaws.com/univaf-loader"
+  image   = "${var.loader_image}"
```
Comment (Mr0grog) [inline terraform/modules/loader/main.tf:5]: Let’s fix the deprecation warning this triggers:

```
Warning: Interpolation-only expressions are deprecated

  on modules/loader/main.tf line 5, in module "loader_task":
   5:   image   = "${var.loader_image}"
```

**Review by Mr0grog — CHANGES_REQUESTED** [review]
Sorry I missed this was waiting on me! Looks good, except for the deprecated syntax. Fix that and let’s get this in!

**Comment by Mr0grog** [comment]
Oh, do we also need to stop pushing a `latest` tag here? https://github.com/usdigitalresponse/univaf/blob/71bf9a488a00d5f80e4b2737d61a2901bd9ebd4e/.github/workflows/ci.yml#L194-L202

**Review by Mr0grog — APPROVED** [review]
👍 👍 👍 Merge at will :shipit: :shipit: :shipit:

**Pull request merged**